test/test_testmon.py

```diff
@@ -192,6 +192,25 @@ def test_add():
         assert tuple(res) == (1, 0, 0), res
         sys.modules.pop('test_a', None)
 
+    def test_tlf(self, testdir):
+        tf = testdir.makepyfile(test_a="""
+            def test_add():
+                1/0
+        """, )
+        testdir.inline_run("--testmon", "-v")
+        sys.modules.pop('test_a', None)
+
+        result = testdir.runpytest("--testmon", "-v")
+        result.stdout.fnmatch_lines([
+            "*1 failed, 1 deselected*",
+        ])
+        tf.setmtime(1424880936)
+        result = testdir.runpytest("--testmon", "-v", "--tlf")
+        result.stdout.fnmatch_lines([
+            "*1 failed in*",
+        ])
```

Comment:
That's the main use case, but I think this is a better minimal test case. We execute identical, failing, test 3 times. 1. test - > creating fixture, 2. test -> we verify that it was deselected because it didnt' change, 3. test -> we verify it was executed (not deselected) because of the --tlf switch, even when not changed.